### ISSUE TITLE
Docker credential helpers not supported on Windows

### ISSUE THREAD

**Issue by rnorth** [testcontainers/testcontainers-java#756]
We've not yet had a chance to test RegistryAuthLocator on Windows, so it is unsupported. 

If we can test it/fix it, it will enable us to properly support this feature.

**Comment by kiview** [comment]
I'll try to check this on Windows this weekend :+1:

**Comment by kiview** [comment]
Here are my results so far. I was logged into docker hub.
For docker hub:
```
22:51:20.108 INFO  docker[alpine:latest] - Pulling docker image: alpine:latest. Please be patient; this may take some time but only needs to be done once.
22:51:20.109 DEBUG org.testcontainers.utility.RegistryAuthLocator - Looking up auth config for image: alpine:latest
22:51:20.110 DEBUG org.testcontainers.utility.RegistryAuthLocator - RegistryAuthLocator has configFile: C:\Users\Kivie\.docker\config.json (exists) and commandPathPrefix: 
22:51:20.112 DEBUG org.testcontainers.utility.RegistryAuthLocator - Executing docker credential helper: docker-credential-wincred to locate auth config for: 
22:51:20.175 ERROR org.testcontainers.utility.RegistryAuthLocator - Failure running docker credential helper (docker-credential-wincred)
22:51:20.177 ERROR org.testcontainers.utility.RegistryAuthLocator - Failure when attempting to lookup auth config (dockerImageName: alpine:latest, configFile: C:\Users\Kivie\.docker\config.json. Falling back to docker-java default behaviour
org.zeroturnaround.exec.InvalidExitValueException: Unexpected exit value: 1, allowed exit values: [0], executed command [docker-credential-wincred, get], output was 26 bytes:
no credentials server URL
```

For quay.io:
```
22:51:54.548 INFO  docker[quay.io/testcontainers/ryuk:latest] - Pulling docker image: quay.io/testcontainers/ryuk:latest. Please be patient; this may take some time but only needs to be done once.
22:51:54.549 DEBUG org.testcontainers.utility.RegistryAuthLocator - Looking up auth config for image: quay.io/testcontainers/ryuk:latest
22:51:54.549 DEBUG org.testcontainers.utility.RegistryAuthLocator - RegistryAuthLocator has configFile: C:\Users\Kivie\.docker\config.json (exists) and commandPathPrefix: 
22:51:54.549 DEBUG org.testcontainers.utility.RegistryAuthLocator - Executing docker credential helper: docker-credential-wincred to locate auth config for: quay.io
22:51:54.604 ERROR org.testcontainers.utility.RegistryAuthLocator - Failure running docker credential helper (docker-credential-wincred)
22:51:54.605 ERROR org.testcontainers.utility.RegistryAuthLocator - Failure when attempting to lookup auth config (dockerImageName: quay.io/testcontainers/ryuk:latest, configFile: C:\Users\Kivie\.docker\config.json. Falling back to docker-java default behaviour
org.zeroturnaround.exec.InvalidExitValueException: Unexpected exit value: 1, allowed exit values: [0], executed command [docker-credential-wincred, get], output was 41 bytes:
credentials not found in native keychain
	at org.zeroturnaround.exec.ProcessExecutor.waitFor(ProcessExecutor.java:1118)
	at org.zeroturnaround.exec.ProcessExecutor.execute(ProcessExecutor.java:925)
	at org.testcontainers.utility.RegistryAuthLocator.runCredentialProvider(RegistryAuthLocator.java:162)
	at org.testcontainers.utility.RegistryAuthLocator.authConfigUsingStore(RegistryAuthLocator.java:129)
	at org.testcontainers.utility.RegistryAuthLocator.lookupAuthConfig(RegistryAuthLocator.java:88)
	at org.testcontainers.images.RemoteDockerImage.resolve(RemoteDockerImage.java:99)
	at org.testcontainers.images.RemoteDockerImage.resolve(RemoteDockerImage.java:27)
...
```

TBH, I not even getting `docker-credential-wincred` working by hand. `list` command works fine, `get` command doesn't return any output for me, maybe I'm doing something wrong?

**Comment by rnorth** [comment]
Ah, we released support for this in 1.9.1 (#868) so I shall close.

**Comment by nathantippy** [comment]
Stacktrace on windows 10  with junit-jupiter 1.10.5
"RegistryAuthLocator - Credentials not found for host (quay.io) when using credential helper/store"


[main] INFO org.testcontainers.dockerclient.DockerClientProviderStrategy - Loaded org.testcontainers.dockerclient.NpipeSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
[main] INFO org.testcontainers.dockerclient.DockerClientProviderStrategy - Will use 'okhttp' transport
[main] INFO org.testcontainers.dockerclient.NpipeSocketClientProviderStrategy - Accessing docker with local Npipe socket (npipe:////./pipe/docker_engine)
[main] INFO org.testcontainers.dockerclient.DockerClientProviderStrategy - Found Docker environment with local Npipe socket (npipe:////./pipe/docker_engine)
[main] INFO org.testcontainers.DockerClientFactory - Docker host IP address is localhost
[main] INFO org.testcontainers.DockerClientFactory - Connected to docker: 
  Server Version: 18.09.1
  API Version: 1.39
  Operating System: Docker Desktop
  Total Memory: 6442 MB
[main] INFO org.testcontainers.utility.RegistryAuthLocator - Credentials not found for host (quay.io) when using credential helper/store (docker-credential-wincred)
[tc-okhttp-stream-2088371948] ERROR com.github.dockerjava.core.async.ResultCallbackTemplate - Error during callback
com.github.dockerjava.api.exception.InternalServerErrorException: {"message":"Get https://quay.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}

	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.execute(OkHttpInvocationBuilder.java:276)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.lambda$executeAndStream$1(OkHttpInvocationBuilder.java:292)
	at java.lang.Thread.run(Thread.java:748)
